pkgs/applications/window-managers/picom/picom-jonaburg.nix

```
@@ -0,0 +1,13 @@
+{ picom, lib, fetchFromGitHub }:
+
+picom.overrideAttrs (oldAttrs: rec {
+  pname = "picom-jonaburg";
+  version = "1.0";
```

There's no `1.0` release.
```suggestion
  version = "unstable-2022-03-19";
```